pegasus/sites.v3/code.org/public/hourofcode/overview.haml

```diff
@@ -1,80 +0,0 @@
----
-title: Hour of Code Teacher Resources
-theme: responsive
----
-
-:css
-  .container_responsive { width: 970px }
-
-%link{href: "/css/student.css", rel: "stylesheet"}
```

Comment:
Is any of this being moved around or is all the new code new?

Reply:
All the new code is new. We're essentially deprecating this `overview.haml` page and using this new code.org/hourofcode (`index.haml`) as the new main hoc page on code.org.